test/system/030-run.bats

```diff
@@ -906,6 +906,17 @@ EOF
     fi
 
     user=$(id -u)
+
+    userspec=$(id -un):$(id -g)
+    run_podman run --hostuser=$user --user $userspec --rm $IMAGE sh -c 'echo $(id -un):$(id -g)'
+    is "$output" "$userspec"
```

Comment:
as a general thing it would be nice to add names to each assertion as that makes reading the logs much easier but I won't block over it.